examples/pytorch/object_detection/maskrcnn/quantization/ptq/fx/pytorch/tools/test_net.py

```diff
@@ -159,14 +159,13 @@ def eval_func(q_model):
                                         is_distributed=distributed, is_calib=True)
     cal_dataloader = MASKRCNN_DataLoader(data_loaders_val)
     if args.tune:
-        from neural_compressor.experimental import Quantization, common
-        quantizer = Quantization("./conf.yaml")
-        quantizer.model = common.Model(model,
-                                       **{'prepare_custom_config_dict': prepare_custom_config_dict}
-                                      )
-        quantizer.calib_dataloader = cal_dataloader
-        quantizer.eval_func = eval_func
-        q_model = quantizer.fit()
+        from neural_compressor import quantization
+        from neural_compressor.experimental import common
+        from neural_compressor.config import PostTrainingQuantConfig
+        model = common.Model(model, **{'prepare_custom_config_dict': prepare_custom_config_dict})
+        conf = PostTrainingQuantConfig(approach="static", backend="pytorch_fx", calibration_sampling_size=[50])
```

Comment:
backend semantic should have been changed, pls be aware of this and double confirm with penghui